api/app/frontend.py

```diff
@@ -0,0 +1,51 @@
+""" Serve up the Single Page Application
+"""
+from starlette.applications import Starlette
+from starlette.types import Scope
+from starlette.routing import Route, Mount
+from fastapi.responses import Response
+from fastapi.templating import Jinja2Templates
+from fastapi.staticfiles import StaticFiles
+from fastapi.requests import Request
+from app import config
+
+
+class SPAStaticFiles(StaticFiles):
+    """ Single Page App Static Files.
+    Serves up .(root, or /) whenever a file isn't found.
+    For a single page app using routing, we need to serve up
+    index.html and let the spa figure out what to serve up.
+    NOTE: Ensure html=True is set
+    """
+    async def get_response(self, path: str, scope: Scope) -> Response:
+        # Call the method on the base class.
+        response = await super().get_response(path, scope)
+        # If file not found, try to serve up the root.
+        if response.status_code == 404:
+            response = await super().get_response('.', scope)
+        return response
+
+
+templates = Jinja2Templates(directory='static')
+
+
+async def get_config(request: Request):
+    """ Apply jinja template response to config.js
+    """
+    return templates.TemplateResponse(
+        "config.js",
```

Comment:
I wonder if you need to create `config.js` file under `/static`

Reply:
I suspect I might have to do that just to get the code coverage good.